packages/datadog-instrumentations/src/mocha/main.js

```diff
@@ -469,7 +469,7 @@ addHook({
 // Used to start and finish test session and test module
 addHook({
   name: 'mocha',
-  versions: ['>=8.0.0'],
+  versions: ['>=5.2.0'],
```

Comment:
just a confirmation, you're making the version support lower than it was before (it's usually not a common thing so i'm just pointing it out)

Reply:
oh yeah, I should've mentioned this. This file is not there for <8 so this should be a noop. I changed it because the testing matrix became way more complicated. We should probably fix our testing setup but that's for another PR 😄